.github/dependabot.yml

```diff
@@ -5,3 +5,4 @@ updates:
     labels: []
     schedule:
       interval: "weekly"
+    target-branch: "stable-5.0"
```

Comment:
Is the `dependabot.yml` file actually being watched in non default branches? My guess would be there has to be another entry in the `main` branches `dependabot.yml` file:

```yaml
version: 2
updates:
  - package-ecosystem: "github-actions"
    ...
  - package-ecosystem: "github-actions"
    directory: "/"
    labels: []
    schedule:
      interval: "weekly"
    target-branch: "stable-5.0"
```

Unfortunately the docs only say

> You must store this file in the .github directory of your repository in the default branch (https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#about-the-dependabotyml-file)

Reply:
Yep, I suspected this wouldn't work and indeed it does not. However your proposed solution does work as I found some other examples in the wild, thanks!!